### PR TITLE
ADD: Like button system test

### PR DESCRIPTION
**Pull request by VladimirMikulic**
Part of #5316

@Uzay-G could you take a look at this? Thanks.

### CODE REVIEW

**Comment by Uzay-G** [comment]
@VladimirMikulic I am busy right now but will check it out in 20 mins. It look good :+1:

File: test/system/post_test.rb [hunk 1]
```diff
@@ -26,13 +26,29 @@ class PostTest < ApplicationSystemTestCase
...
-      
+
     end
 
   end
 
+  test 'like button on the post' do
+    visit '/wiki/wiki-page-path/comments'
+
+    find('a[data-target="#loginModal"]', text: 'Login').click()
```
Comment (Uzay-G) [inline test/system/post_test.rb]: It's a bit more readable here to just write:
`click_on "Login"`

Comment (VladimirMikulic) [reply test/system/post_test.rb]: Not really, this will trigger the modal, `click_on "Login"` doesn't work.

Comment (Uzay-G) [reply test/system/post_test.rb]: It works for me. Clicking the button automatically triggers the modal

Comment (VladimirMikulic) [reply test/system/post_test.rb]: Let's not make Travis angry, it's readable enough :)

**Comment by SidharthBansal** [comment]
I will review after Uzay reviews it.
Kind of busy with lunch.

**Comment by SidharthBansal** [comment]
You can write more tests. We can evaluate marks after merging all the prs. @Uzay-G I can't see your system tests prs. It will be great if you can start with system tests too. Thanks both

**Comment by Uzay-G** [comment]
> You can write more tests. We can evaluate marks after merging all the prs. @Uzay-G I can't see your system tests prs. It will be great if you can start with system tests too. Thanks both

I have one at https://github.com/publiclab/plots2/pull/7254 and another one was merged at https://github.com/publiclab/plots2/pull/7251

**Comment by jywarren** [comment]
Trying to bypass the codecov issue...

**Comment by SidharthBansal** [comment]
System test 2 task's test 1. Trying to get 4 more tasks to get your system test 2 approved too

**Comment by VladimirMikulic** [comment]
@SidharthBansal you can count on me :)

**Comment by SidharthBansal** [comment]
Yeah I am counting you. Policies are same for all students. Sorry if anything unfair or anything wrong I did

**Comment by VladimirMikulic** [comment]
Why are you apologising? You are one of the best mentors of this GCI.
Let's be real you were with us even when you were on vacation and while you had a fever...

I appreciate that very much and I apologise if I had said something wrong or did something work.

**Comment by SidharthBansal** [comment]
I just thought you felt that I am not treating all the gci students equally. You have not done anything wrong :-)

**Comment by SidharthBansal** [comment]
You all are really hardworking. If I was that much hardworking at your age the I would be in the skies today. :-)

**Comment by VladimirMikulic** [comment]
You are a Google Tech Intern. It means that you are hardworking.

**Comment by Uzay-G** [comment]
This is a nice thread! Thanks for mentoring us @SidharthBansal.
I will get started on all the issues you mentioned this weekend once I have
more time :+1:

**Comment by SidharthBansal** [comment]
I was. I was rejected there.

**Comment by VladimirMikulic** [comment]
@SidharthBansal I completely agree with @Uzay-G.
You really have great `$ uptime` and you should be proud of yourself :)
I've learned a lot of great things from you.

**Comment by SidharthBansal** [comment]
Thanks

**Comment by jywarren** [comment]
sending ❤️  for this convo! 🎉

**Comment by SidharthBansal** [comment]
Let's complete System tests this GCI.
I was thinking of Project-Based Mentorship in GCI which I proposed https://github.com/publiclab/plots2/issues/3285 but there was not wide momentum at that time. Last year, @jywarren we did the log in modal and this year we are doing System Tests through this. I think we should reopen this issue and escalate the discussion. 
The advantages are HUGE. The problem is simple. We require DEDICATED MENTORSHIP. One ongoing work is Code-based rights. When we will club these two things (Project based DEDICATED Mentorship in GCI + Code Based rights(https://github.com/publiclab/plots2/pull/7271)), the GCI ladder will be better than GSoC ladder. Also, I will say, it could result in early talent hunting for Public Lab. 
If you think we can go with https://github.com/publiclab/plots2/issues/3285 from next GCI, we can plan in that direction too in February. Please feel free to reopen the issue.

Suggestions needed 
Thanks 
Sidharth

**Comment by VladimirMikulic** [comment]
Wow, it is a really cool idea. We should first define what needs to be done and then split it into smaller issues.

**Comment by SidharthBansal** [comment]
This is a really EPIC thing to discuss. I will escalate it at the end of GCI

**Pull request merged**